docs/api-docs/combineReducers.md

```diff
@@ -0,0 +1,27 @@
+# `combineReducers(reducersMap)`
+
+* `reducersMap: Object<string, ReducerFunction>` &ndash; a map of keys to nested
+  reducers, just like the `combineReducers` you would find in Redux itself.
+
+## Notes
+
+Reducer composition is key to a clean Redux application. The built-in Redux
+`combineReducers` won't work for nested reducers that use `loop`, so we included
+one that is aware that some reducers might have side effects. The `combineReducers`
+in redux-loop knows how to compose cmds as well as state from nested reducers
+so that your effects tree is always separate from your state tree. It's also
```

Comment:
I've always felt that this line is confusing. 

It's also completely compatible with the one in Redux, so there should be no issue switching to this implementation.

I think we should reword it or get rid of it. I think it was intended to point out that if something out of your control is using vanilla combineReducers and has no loops inside of it, then you can pass the result of that to the redux loop version. But the wording just seems confusing to me.

Reply:
or do you think it would read better to have a short description above and the detailed notes below (where they are now)